src/index.html

```diff
@@ -7,6 +7,39 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar">
```

Comment:
Add some modifiers that you use for your calendar so the reviewer can test it.

Reply:
@cty2802  Please correct the error described in the previous comment.